package.json

```diff
@@ -9,54 +9,42 @@
     "sitemap.xml",
     "seo"
   ],
-  "repository": "https://github.com/nuxt-community/sitemap-module",
   "license": "MIT",
   "contributors": [
-    {
-      "name": "Nicolas Pennec"
-    },
-    {
-      "name": "Pooya Parsa"
-    }
-  ],
-  "files": [
-    "lib"
+    "Nicolas Pennec",
+    "Pooya Parsa <pooya@pi0.ir>"
   ],
   "main": "lib/module.js",
+  "repository": "https://github.com/nuxt-community/sitemap-module",
+  "publishConfig": {
+    "access": "public"
+  },
```

Comment:
I use `npx npm sort-package-json` to reorder the package.json

for eg., it will push the `publishConfig` to the end of file.